### DISCUSSION TITLE
Are you a power user of Wikis? I want to hear from you!

### DISCUSSION THREAD

**Discussion by shmeadyy** [community/community#21088] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey GitHub community! We're looking to learn more about how you use wikis to share information with other developers about your project. Do you have a highly utilized wiki page? Do you put all of your documentation there?  Or do you use it to flush out ideas about the project before it even begins?

I'd like to hear from you and I'd love to see examples of how you use Wikis!  I'd also love to hear some feature requests (like [this one](https://github.com/github-community/community/discussions/11612) for Mermaid integration support).

Please share some links of wikis that you have either created or that you leverage as a user below, and I'll reach out directly to schedule some time to chat. Thanks for helping us make GitHub great!

**Comment by bisdevres** [comment]
i'm becoming a GitHub wiki power user

we're migrating from Azure DevOps to GitHub enterprise cloud.
we notice that github has a less advanced wiki, compared to azure devops.
in articular we're **missing these 3 features**:

1. [Publish a Git repository to a wiki](https://docs.microsoft.com/en-us/azure/devops/project/wiki/publish-repo-to-wiki?view=azure-devops&tabs=browser#publish-a-git-repository-to-a-wiki-1)
2. [Change the page sequence, add, or update a .order file](https://docs.microsoft.com/en-us/azure/devops/project/wiki/publish-repo-to-wiki?view=azure-devops&tabs=browser#page-sequence)
3. [have folder structure in the table of content](https://docs.microsoft.com/en-us/azure/devops/project/wiki/publish-repo-to-wiki?view=azure-devops&tabs=browser#promote-folder-to-page)

note: i can't show you any wiki examples (as they are part of a private enterprise).

cheers

**Reply by shmeadyy** [reply]
amazing - thank you for sharing @bisdevres!

**Comment by lanej0** [comment]
We use the wiki for publishing all of the testing instructions for [each release of WooCommerce](https://github.com/woocommerce/woocommerce/wiki) as well as keeping a good amount of documentation in there. Does that qualify us as a power user?

**Reply by shmeadyy** [reply]
yes i'd say so! would you be open to having a chat?

**Reply by lanej0** [reply]
@shmeadyy Sure.  I'm out on vacation for a week and a half as of Thursday.  Is that too long of a wait for you?

**Reply by shmeadyy** [reply]
Nope - that's fine! Feel free to set up some time on my calendar! https://calendly.com/laurathorson/wiki-user-research

**Reply by lanej0** [reply]
Done!

**Comment by danielhjacobs** [comment]
The Ruffle wiki (https://github.com/ruffle-rs/ruffle/wiki) has a lot of useful information about features, usage, FAQs, and oddities relating to Ruffle and Adobe Flash Player. We got a lot of useful contributions from the general public, especially on the "Helpful Resources" page.

However, a few spam bots made "contributions" by replacing legitimate links with malware, and since we don't want the organization's wiki becoming a malware distributor, we had no choice but to make the wiki collaborators only.

Edit: The wiki has been switched back to being editable by anyone, but with [a hook to notify us on Discord when the Wiki is edited](https://github.com/oznu/gh-wiki-edit-discord-notification), but we're not sure how sustainable that will be.

So in terms of a feature request, there are a few:

1) Allow users to open "PRs" against a collaborators only wiki. They can then be merged like normal PRs by collaborators.
2) Add a less restrictive "contributors only" setting. This would still keep spambots from adding malware without removing all helpful community contributions.
3) The prior suggestion here: https://github.com/community/community/discussions/10376

**Reply by n0samu** [reply]
Another update: the `gollum` event seems not trigger when a wiki page is *deleted* rather than *edited* (https://github.com/community/community/discussions/39550), but deletions are even more important for us to know about! We've thus resorted to using a [Google Apps Script](https://github.com/n0samu/gh-wiki-monitor) to periodically check our wiki's history page, since any solution based on GitHub's API or Actions seems to be insufficient for monitoring all changes to our wiki.

**Comment by shmeadyy** [comment, marked as answer]
Thank you all for the conversations (either over Zoom or async here in the discussion)! We've made some decisions about the direction of wikis - stay tuned in the future!
